packages/app/src/runner/aut-iframe.ts

```diff
@@ -93,7 +93,7 @@ export class AutIframe {
    * Otherwise, if top and the AUT match origins, the method returns true.
    * If the AUT origin is "about://blank", that means the src attribute has been stripped off the iframe and is adhering to same origin policy
    */
-  doesAUTMatchTopOriginPolicy = () => {
+  doesAUTMatchTopSuperOriginPolicy = () => {
```

Comment:
Can you add definitions for these changes? When is it top origin vs super origin vs super domain origin?

Off hand not sure of the difference here & wondering why/if we need to expose this in console log props to the tester.

Reply:
Yeah, i was going to mention that, we need a checked in list of names and what they mean.

Here's a few terms we should probably define, maybe more.

origin
originPolicy
superOriginPolicy
superDomainOriginPolicy

Reply:
@mjhenkes I am thinking the three as specific to generic, something like:

```
origin : https://www.foobar.com:443/path/to/resource.ext
originPolicy : https://www.foobar.com:443
superDomainOriginPolicy : https://foobar.com:443
```

I'm just wondering where a good place to document this is?

Reply:
in our current terms, isn't origin === originPolicy?

given url https://www.foobar.com:443/path/to/resource.ext

wouldn't location.origin = https://www.foobar.com:443 ?

Reply:
I'm happy enough with `superDomainOriginPolicy` but i think we can drop `originPolicy` altogether

Reply:
Ah I think you're right. I think we are conflating origin with policy. It's just it's origin in this case. In that case shouldn't it be `superDomainOrigin`? and then the comparing function are checking for `sameSuperDomainOrigin` or  `sameSuperDomainOriginPolicy` or `sameOrigin` or `sameOriginPolicy`. I like `isSameSuperDomainOrigin` and `isSameOrigin`, since you can infer if the policy matches if they are the same or not.

Reply:
I'm good with dropping `policy` from the names.

Reply:
so something like:

origin: `https://www.foobar.com:443`
superDomainOrigin:` https://foobar.com:443`

isSameOrigin: `https://www.foobar.com:443/path/to/resource.ext` and `https://www.foobar.com:443/path/to/something/else.ext` is `true`
isSameOrigin: `https://www.foobar.com:443/path/to/resource.ext` and `https://app.foobar.com:443/path/to/something/else.ext` is `false`
isSameOrigin: `https://www.foobar.com:443/path/to/resource.ext` and `https://foobar.com:443/path/to/something/else.ext` is `false`

isSameSuperDomainOrigin: `https://www.foobar.com:443/path/to/resource.ext` and `https://www.foobar.com:443/path/to/something/else.ext` is `true`
isSameSuperDomainOrigin: `https://www.foobar.com:443/path/to/resource.ext` and `https://app.foobar.com:443/path/to/something/else.ext` is `true`
isSameSuperDomainOrigin: `https://www.foobar.com:443/path/to/resource.ext` and `https://foobar.com:443/path/to/something/else.ext` is `true`

Reply:
Yep, that all looks solid to me.

Reply:
Perfect. Unless there are objections, I am going to rename all the things

Reply:
All the things are renamed if anyone wants to take a second look